charts/system/kubelet-csr-approver/values.yaml

```diff
@@ -1,18 +1,122 @@
 image:
-  repository: tccr.io/tccr/alpine
+  repository: ghcr.io/postfinance/kubelet-csr-approver
```

Comment:
afaik postfinance sint the official source of this?

Reply:
Can't find any other source on GitHub or otherwise.

Reply:
you're correct, nvm